okhttp/src/test/java/okhttp3/EventListenerTest.java

```diff
@@ -115,6 +114,15 @@
     if (cache != null) {
       cache.delete();
     }
+
+    client.connectionPool().evictAll();
+    if (client.connectionPool().connectionCount() > 0) {
```

Comment:
Move this to `OkHttpClientTestRule.ensureAllConnectionsReleased()`

Reply:
... actually it has an assertion for this. Maybe replace that assertion with this?